### ISSUE TITLE
Choosing Functionblocks from the UI on model creation is broken

### ISSUE THREAD

**Issue by kolotu** [eclipse/vorto#2709]
<img width="1792" alt="Screenshot 2020-11-19 at 08 37 22" src="https://user-images.githubusercontent.com/56577255/99635663-958a2980-2a42-11eb-91bc-1b0a88192fb0.png">

**Comment by mena-bosch** [comment]
The search query for functionblocks in that wizard step is: `state:Released namespace:org.eclipse.vorto type:Functionblock`. 
After running the query manually (save for the `state` parameter), it seems a lot of those models have been deprecated.
The `lastModifiedBy` field shows your Bosch user as the deprecator. 
Likely, this is just about fine-tuning the search expression in `modelCreationService.js`. 
Note that you can have multiple namespaces, wildcards, etc. etc. in the expression, in case they have been somehow moved in different places.

**Comment by kolotu** [comment]
@mena-bosch good point - the query should be `state:Released namespace:org.eclipse.vorto* type:Functionblock` - the asterisk is required, since we use subnamespaces now.

**Comment by mena-bosch** [comment]
> @mena-bosch good point - the query should be `state:Released namespace:org.eclipse.vorto* type:Functionblock` - the asterisk is required, since we use subnamespaces now.

Sounds like an easy one then ;)
